### PR TITLE
Use new GSI tag to turn on data

### LINKED ISSUE #2096: GFSv16.3.12 - Turn on active assimilation of NOAA-21 radiances
### Description

This operational update will turn on the active assimilation of NOAA-21 radiances. This is a companion update to the previous GFSv16.3.11 upgrade that turned on passive monitoring of NOAA-21 radiances.

### Target version

v16.3.12

### Tasks

- [x] Create release branch (`release/gfs.v16.3.12`)
- [x] New GSI tag (update `Externals.cfg` and `sorc/checkout.sh`
- [x] Update`gfs_ver=v16.3.12` and `version=v16.3.12` in `versions/run.ver` (and mention in release notes)
- [x] Create/add new release notes
- [x] Cut hand-off tag for CDF (`EMC-v16.3.12`)
- [x] Submit CDF to NCO
- [x] Implementation into operations complete
- [x] Merge release branch into operational branch - https://github.com/NOAA-EMC/global-workflow/pull/2167
- [x] Cut version tag from operational branch
- [x] Release new version tag
- [x] Announce to users
- [x] Update Read-The-Docs operations status version in develop - #2169

### PR DESCRIPTION
**Pull request by ADCollard**
This PR turns on active assimilation of NOAA-21 CrIS, NOAA-21 ATMS, GOES-18 Clear Sky Radiances and Metop-C ASCAT.

All substantive changes are confined to the fix directory of GSI and this PR simply checks out the `gfsda.v16.3.12` tag of the GSI.

Resolves #2096 

# How has this been tested?
Clone and build on WCOSS2.  Single cycle test.

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Comment by ADCollard** [comment]
@KateFriedman-NOAA Does the GFS version number get incremented in `versions/run.ver`?  It seems to have been in the past but I am not sure this is correct.

File: docs/Release_Notes.gfs.v16.3.12.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+GFS V16.3.11 RELEASE NOTES
```
Comment (aerorahul) [inline docs/Release_Notes.gfs.v16.3.12.md]:
```suggestion
GFS V16.3.12 RELEASE NOTES
```

**Review by aerorahul — CHANGES_REQUESTED** [review]
small typo.

**Review by aerorahul — APPROVED** [review]

File: docs/Release_Notes.gfs.v16.3.12.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+Lastly, link the ecf scripts by moving back up to the ecf folder and executing:
+```bash
+cd ../ecf
+./setup_ecf_links.sh
+```
+
+Additional release notes for `upp/8.3.0` installation on WCOSS2:
+https://docs.google.com/document/d/18bUYWmsN7FuweyA2CQTBW1rk3FNRMHKPlYHZlbS8ODs/edit?pli=1
```
Comment (KateFriedman-NOAA) [inline docs/Release_Notes.gfs.v16.3.12.md]: These notes were for the prior v16.3.11 upgrade. Am guessing we don't need these in these release notes now.

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good and includes expected changes for this. Thanks @ADCollard !

**Comment by KateFriedman-NOAA** [comment]
@ADCollard Just one comment, forgot to mention I left it when approving this PR. Please address and we can merge it. Thanks!

File: docs/Release_Notes.gfs.v16.3.12.md (hunk 2 above)
Comment (ADCollard) [reply docs/Release_Notes.gfs.v16.3.12.md]: @KateFriedman-NOAA Removed.

**Comment by ADCollard** [comment]
> @ADCollard Just one comment, forgot to mention I left it when approving this PR. Please address and we can merge it. Thanks!

@KateFriedman-NOAA It should be good now

**Pull request merged**